distributor/src/distributor/results.bal

```diff
@@ -23,6 +23,22 @@ import ballerina/websub;
     }
 }
 service receiveResults on resultsListener {
+
+    @http:ResourceConfig {
+        methods: ["POST"],
+        path: "/notification/{electionCode}/{resultCode}"
+    }
+    resource function receiveUpcomingResultNotification(http:Caller caller, http:Request req, string electionCode,
+                                                        string resultCode) returns error? {
+        if validTwilioAccount {
+            _ = start sendSMS(<string> electionCode, <string> resultCode);
```

Comment:
We need more info other than electionCode, resultCode. I actually changes the logic here https://github.com/ECLK/Results-Dist/pull/88/files#diff-e9297e2e2e08adebbe67b7f949f70c37R22